opta/commands/apply.py

```diff
@@ -334,6 +326,39 @@ def _apply(
         )
 
 
+def _verify_semver(old_semver_string: str, current_semver_string: str) -> None:
+    if old_semver_string not in [DEV_VERSION, ""] and current_semver_string not in [
```

Comment:
nit: a slightly cleaner way is if you check `if old_semver_string in [DEV_VERSION, ""]...; return` Then you won't have to indent the whole block below this

Reply:
done